code/modules/mob/living/carbon/human/human.dm

```diff
@@ -408,18 +408,117 @@
 /mob/living/carbon/human/proc/get_idcard()
 	if(wear_id)
 		return wear_id.GetID()
-
-//Removed the horrible safety parameter. It was only being used by ninja code anyways.
-//Now checks siemens_coefficient of the affected area by default
+//now more shit and less good but now shocks more things
 /mob/living/carbon/human/electrocute_act(var/shock_damage, var/obj/source, var/base_siemens_coeff = 1.0, var/def_zone = null, var/tesla_shock = 0)
 	if(status_flags & GODMODE)	return 0	//godmode
-
-	if (!def_zone)
-		def_zone = pick("l_hand", "r_hand")
-
+	if (!def_zone) //if no Defined zone.
+		switch (rand(1,6))// These go through 5 different "arcs", scaling down the given damage value per limb effeccted. so the foot will never get as damaged as the hand, ideally.

```

Comment:
This does more damage now, as it's also referring to parent, FYI.

Also, please write this in a more sane way:
```
if (!def_zone)
    var/list/damage_areas = list()
    switch (rand(1, 6))
        if (1)
            damage_areas = list("l_hand", "l_arm", "chest", "r_arm")
        if (2)
            damage_areas = list("bl", "ble", "Blo")
        // so on

    if (gloves)
        shock_damage *= gloves.siemens_coefficient

    for (var/area in damage_areas)
        apply_damage(sock_damage, BURN, area)
        shock_damage *= 0.8

    visible_message()
```

I guess you lose 1 special siemes_coeff check but. Eh. This amount of verbose code is just bad, IMO.